tests/unitTests/ArticleReducer.test.jsx

```diff
@@ -0,0 +1,32 @@
+import articleReducer from "../../src/reducers/articleReducer";
+import { POST_ARTICLE } from "../../src/actions/types";
+
+describe("socialLoginReducer", () => {
+  it("has a default state", () => {
+    expect(articleReducer(undefined, { type: "unexpected" })).toEqual({
+      article: {},
+      articles: []
+    });
+  });
+
+  it("updates state on POST_ARTICLE action type", () => {
+    expect(
+      articleReducer(undefined, {
+        type: POST_ARTICLE,
+        payload: { article: {}}
+      })
+    ).toEqual({
+      article: {article: {}}, articles: []
+    });
+  });
+  it("updates state on POST_ARTICLE action type", () => {
```

Comment:
Could you remove this duplicate file as it is similar to the one above

Reply:
@shakirandagire I have worked on this